### ISSUE TITLE
Multi-Select Form Dropdowns Not Selecting Correctly

### ISSUE THREAD

**Issue by sky-gregory** [gunjandatta/gd-bs#35]
When selecting or deselecting values, the control does not behave as expected. There almost appears to be multiple click events. 

```javascript
control: {
    label: "Testing",
    name: "Example",
    type: Components.FormControlTypes.MultiDropdown,
    items: [
        {
            text: "Net Estimated Hard",
            value: "EstHard",
            isSelected: _financialValuesParam.indexOf("EstHard".toLowerCase()) >= 0 || _financialValuesParam === ""
        },
        {
            text: "Net Estimated WCF",
            value: "EstWCF",
            isSelected: _financialValuesParam.indexOf("EstWCF".toLowerCase()) >= 0 || _financialValuesParam === ""
        },
        {
            text: "Net Estimated NAF",
            value: "EstNAF",
            isSelected: _financialValuesParam.indexOf("EstNAF".toLowerCase()) >= 0 || _financialValuesParam === ""
        },
        {
            text: "Net Programmed/Budgeted",
            value: "PB",
            isSelected: _financialValuesParam.indexOf("PB".toLowerCase()) >= 0 || _financialValuesParam === ""
        },
        {
            text: "Net Actual WCF",
            value: "ActWCF",
            isSelected: _financialValuesParam.indexOf("ActWCF".toLowerCase()) >= 0 || _financialValuesParam === ""
        },
        {
            text: "Opportunity Cost",
            value: "OppCost",
            isSelected: _financialValuesParam.indexOf("OppCost".toLowerCase()) >= 0 || _financialValuesParam === ""
        },
    ],
    onValidate: control => {
        let allValues = _form.getValues();

        //Get the selected Bins
        let selectedValues = allValues["Example"].map(item => item.data);

        let isValid = selectedValues.length > 0;

        //If there is a value, then the control is valid
        return {
            invalidMessage: "At least one Financial Value must be selected",
            isValid
        };
    }
} as Components.IFormControlPropsDropdown
```

**Comment by gunjandatta** [comment]
The "name" property doesn't match:

FinancialValues vs Example

**Comment by sky-gregory** [comment]
That was me trying to update for pasting as an example. In the code, they match.

**Comment by gunjandatta** [comment]
The "value" is an array of the selected items. I see the selected items working as designed. What exactly are you seeing?

**Comment by gunjandatta** [comment]
Note - Remove click events from v4, as they are no longer needed in v5.

**Comment by gunjandatta** [comment]
Fixed in v3.2.7
